package.json

```diff
@@ -7,7 +7,7 @@
     "url": "git://github.com/mxstbr/react-boilerplate.git"
   },
   "engines": {
-    "npm": ">=5"
+    "npm": ">=3"
```

Comment:
Agreed, but should be "node": ">=4". Reason: if Node 4 is to be deprecated other areas need to be adjusted as well (travis and appveyor configs, docs, other?). These changes are out of scope for this PR. I guess it could be argued that `engines: node` could even be out of scope.